apps/qubit/modules/informationobject/templates/multiFileUploadSuccess.php

```diff
@@ -62,6 +50,12 @@
 
           <legend><?php echo __('Import multiple digital objects') ?></legend>
 
+          <div class="alert alert-info">
+            <p><?php echo __('Add your digital objects by dragging and dropping local files into the pane below, or by clicking the browse link to open your local file explorer.') ?></p>
+            <p><?php echo __('The Title and Level of description values entered on this page will be applied to each child description created for the associated digital objects - %dd% represents an incrementing 2-value number, so by default descriptions created via this uploader will be named image 01, image 02, etc.') ?></p>
```

Comment:
Is `%dd%` going to be an issue with the translations? I think the %whatever% pattern is used to replace content.

Reply:
I have exported the XLIFF with this '%dd%' string included and it does not seem to cause any issues with extracting the string. I have added single quotes around '%dd%' for clarity when reading.

```
<trans-unit id="906116b1dd3e40cf3c978815d31c78ba03e1f034">
        <source>The Title and Level of description values entered on this page will be applied to each child description created for the associated digital objects - '%dd%' represents an incrementing 2-value number, so by default descriptions created via this uploader will be named image 01, image 02, etc.</source>
        <target/>
        <note>https://github.com/artefactual/atom/blob/master/apps/qubit/modules/informationobject/templates/multiFileUploadSuccess.php</note>
      </trans-unit>
```